extensions/ion/ion-core/src/main/java/org/eclipse/dataspaceconnector/ion/spi/request/InputValidator.java

```diff
@@ -9,10 +9,6 @@
 
 public class InputValidator {
     public static void validateEs256kOperationKey(JWK key, String visibility) {
-        var allowedProperties = new HashSet<>(Set.of("kty", "crv", "x", "y"));
```

Comment:
Just out of curiosity: why did CodeQL complain here?

Reply:
Here the alert was: [**Container contents are never accessed**](https://codeql.github.com/codeql-query-help/java/java-unused-container/)